cli/flox-rust-sdk/src/models/manifest/typed.rs

```diff
@@ -61,15 +61,7 @@ macro_rules! impl_into_inner {
 
 // We use skip_serializing_if throughout to reduce the size of the lockfile and
 // improve backwards compatibility when we introduce fields.
-// We don't use Option and skip_serializing_none because an empty table gets
-// treated as Some,
-// but we don't care about distinguishing between a table not being present and
-// a table being present but empty.
-// In both cases, we can just skip serializing.
-// It would be better if we could deny_unknown_fields when we're deserializing
-// the user provided manifest but allow unknown fields when deserializing the
-// lockfile,
-// but that doesn't seem worth the effort at the moment.
```

Comment:
nit: I think this part of the comment is still relevant?

Reply:
Ah yes, I thought it was part of the same paragraph, but it's unrelated.